Assets/Plugins/iOS/Models/TargetingParameters+BidMachine.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  TargetingParameters+BidMachine.swift
+//  UnityFramework
+//
+//  Created by Dzmitry on 30/09/2024.
+//
+
+import Foundation
+import BidMachine
+import CoreLocation
+
+extension TargetingParameters.Gender {
+    var asBMGender: BidMachine.UserGender {
```

Comment:
You can reverse logic and make init for `BidMachine.UserGender` from `TargetingParameters.Gender`

Reply:
✅